gnomad/utils/intervals.py

```diff
@@ -112,3 +112,27 @@ def _add_padding(
         return [_add_padding(i) for i in intervals]
     else:
         return _add_padding(intervals)
+
+
+def explode_intervals_to_loci(
+    ht: hl.Table,
+    keep_intervals: bool = False,
+) -> hl.Table:
+    """
+    Expand intervals to loci.
+
+    :param ht: Hail Table with an interval field.
+    :param keep_intervals: If True, keep the original interval as a column in output.
+    :return: Hail Table keyed by loci and intervals as optional field.
+    """
+    ht = ht.annotate(
+        pos=hl.range(ht.interval.start.position, ht.interval.end.position + 1),
```

Comment:
this should double check whether the interval includes the start and end positions and adjust accordingly